tutorials/2d/2d_movement.rst

```diff
@@ -60,6 +61,46 @@ Add a script to the kinematic body and add the following code:
         get_input()
         move_and_slide(velocity)
 
+ .. code-tab:: csharp
+
+    using Godot;
+    using System;
+
+    public class Movement : KinematicBody2D
+    {
+        [Export] public int Speed = 200;
+
+        Vector2 velocity = new Vector2();
+
+        public void _GetInput()
```

Comment:
I think you forgot this, as we just discuss about it on Discord :)

Reply:
Make `velocity` upper case?

Reply:
No, I meant `_Input`, sorry! It seems that you changed the name already in the following snippet but forgot to do it here.